### ISSUE TITLE
Post to twitter on publishing blog entry

### ISSUE THREAD

**Issue by glaslos** [honeynet/honeynet.github.io#54]
Integration between GitHub and Twitter/Bluesky/...

**Comment by Sneharaj7645653** [comment]
Hello,
I would like to work on this issue. Could you please assign it to me?

Thank you!

**Comment by hgascon** [comment]
@glaslos what exactly do you want to do?

**Comment by glaslos** [comment]
I was thinking we could setup a GitHub action that posts on social media on a new blog post.

**Comment by nidhaahmed** [comment]
Hey @hgascon, I am thinking of solving this issue by configuring a GitHub Action to detect when a new blog entry is published. Upon detection, the workflow triggers a script to post to Twitter via the Twitter API. 

- We can start by setting up access to Twitter’s API, which means creating a Twitter Developer account to get the necessary API keys. 
- Once we have those, we can safely store them as secrets in the GitHub repository. 
- From there, we can set up a workflow in GitHub Actions. This workflow can automatically detect when a new blog post is published and run a script to send out a tweet with the post’s title and link.
- Hence manual tweeting won't be necessary.

Can you assign this issue to me so I can implement the solution.

**Comment by hgascon** [comment]
@glaslos What do you think?

**Comment by nidhaahmed** [comment]
@hgascon , can I start working on it?

**Comment by nidhaahmed** [comment]
@hgascon , can I start working on it?

**Comment by glaslos** [comment]
Yes, that sounds like a very reasonable approach. @nidhaahmed please prepare a pull request.
